<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSession</class>
 <widget class="QWidget" name="VSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_session">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="VConnection" name="widget_vconnect" native="true"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>1</number>
        </property>
        <widget class="VReceive" name="widget_vreceive" native="true"/>
        <widget class="VSend" name="widget_vsend" native="true"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VReceive</class>
   <extends>QWidget</extends>
   <header>vreceive.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VSend</class>
   <extends>QWidget</extends>
   <header>vsend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VConnection</class>
   <extends>QWidget</extends>
   <header>vconnection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
